compiler-rt/lib/sanitizer_common/sanitizer_linux.cpp

```diff
@@ -1237,6 +1242,12 @@ uptr GetPageSize() {
   CHECK_EQ(rv, 0);
   return (uptr)pz;
 #    elif SANITIZER_USE_GETAUXVAL
+
+#      if SANITIZER_ANDROID && __ANDROID_API__ < 35
+  if (!strerrorname_np)
```

Comment:
Leave comment about the API level this was introduced by, and also add the fact that before that API level, the page size was always 4096

Reply:
This is missing that `strerrorname_np` was also introduced in Android 15 (reference is https://cs.android.com/android/platform/superproject/main/+/main:bionic/libc/libc.map.txt?q=strerrorname_np%20f:bionic%2Flibc%2Flibc.map.txt)